speakerscorner.md

```diff
@@ -0,0 +1,10 @@
+# Speaker's Corner
+
+Named after the renowned corner of Hyde Park in London, the Speaker's Corner seminars are meant to provide a platform
```

Comment:
```suggestion
Named after the renowned corner of Hyde Park in London, the Speaker's Corner seminars are a platform
```

Reply:
Thanks for the updates. This was meant as a quick draft just to get a page started. But I'm happy to start tweaking from here regardless 👍